### DISCUSSION TITLE
change the encode

### DISCUSSION THREAD

**Discussion by nguyennhi263** [spatie/simple-excel#165] [General]
Can i change the encode style
ex: Shift_JIS

**Comment by d8y** [comment]
Are there any discussions underway to change encode? 
I would like this feature too, so can I implement it and submit a PR?

**Reply by nguyennhi263** [reply]
> Are there any discussions underway to change encode? I would like this feature too, so can I implement it and submit a PR?

I encode before using lib
It's very usefull if have that implement
